helm/reana/templates/reana-server.yaml

```diff
@@ -93,6 +93,11 @@ spec:
               secretKeyRef:
                 name: {{ include "reana.prefix" . }}-cern-gitlab-secrets
                 key: REANA_GITLAB_HOST
+          - name: REANA_SECRET_KEY
```

Comment:
Do I need to add this env also in `reana-workflow-controller.yaml`? I see there are other cases in both templates such us `REANA_DB_USERNAME`.

Reply:
In principle, we do not access `user.access_token` anywhere else than in RS, if SQLAlchemy doesn't need it (does lazy loading) we can skip adding it there (the fewer places we have this key loaded the best from a security point of view). I would say it needs testing.

Reply:
How would you test it?